src/trace-plugin-loader.js

```diff
@@ -49,6 +49,19 @@ function checkLoadedModules() {
   }
 }
 
+function checkPatch(patch) {
+  if (!patch.patch && !patch.intercept) {
+    throw new Error('Plugin patch for ' + patch + ' doesn\'t patch ' +
```

Comment:
This message could be ambiguous. Maybe "Plugin for ' + patch + ' doesn\'t contain any patches."?